### PR TITLE
PyHive version should include pretty printing of structs

### LINKED ISSUE #2591: PyHive version should include pretty printing of structs
Copies https://github.com/mozilla/redash/issues/264

Creating issue to submit PR against.

### PR DESCRIPTION
**Pull request by alison985**
fixes #2591 and https://github.com/mozilla/redash/issues/264

There was a lot of conversation on how best to solve this in https://github.com/mozilla/redash/issues/264 but this is where we ended up for a short-term solution. Any issues?

### CODE REVIEW

File: redash/query_runner/presto.py [hunk 1]
```diff
@@ -95,7 +159,7 @@ def get_schema(self, get_stats=False):
         return schema.values()
 
     def run_query(self, query, user):
-        connection = presto.connect(
+        connection = presto.pretty_connect(
```
Comment (arikfr) [inline redash/query_runner/presto.py]: Shouldn't this be just `pretty_connect(...)`?

**Comment by alison985** [comment]
Yes, it's just nicer output.

Great catch on the function call. I've modified.

**Comment by jezdez** [comment]
The amount of code this adds just for nicer output still freaks me out, IMO way too much copied code from PyHive to be sensible to maintain for Redash, see the discussion in https://github.com/mozilla/redash/issues/264 for more info.

In the meantime, I've opened https://github.com/dropbox/PyHive/pull/222 and say we wait for this to land first before we move on. (I'll fix the tests there in the coming days)

**Comment by jezdez** [comment]
We discussed this with @arikfr in person and decided to move this into an own Redash extension in redash-stmo.

**Pull request closed without merging**